### PR TITLE
v4l2rtspserver: add package

### PR DESCRIPTION
**Pull request by roger-**
Signed-off-by: Roger D <rogerdammit@gmail.com>
f62794a

Maintainer: me
Compile tested: ramips/mt7620 LEDE 17.01.4 + snapshot
Run tested: ramips/mt7620 LEDE 17.01.4

Description: V4l2 RTSP server

New pull with the right package name and some other fixes.

### CODE REVIEW

**Comment by roger-** [comment]
@hnyman: could you please take a look at this?

**Comment by thess** [comment]
It would also be nice if you could fix your git user-name to match your SoB name and use your full name, please.

Builds OK - will test on a live system shortly.

File: multimedia/v4l2rtspserver/files/config.lede [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+OBJ =				o
+LINK =				$(CXX) -o
+LINK_OPTS =			-L. $(LDFLAGS)
+CONSOLE_LINK_OPTS =		$(LINK_OPTS)
+LIBRARY_LINK =			$(AR) cr 
+LIBRARY_LINK_OPTS =    
+LIB_SUFFIX =			a
+LIBS_FOR_CONSOLE_APPLICATION =	$(CXXLIBS)
+LIBS_FOR_GUI_APPLICATION =	$(LIBS_FOR_CONSOLE_APPLICATION)
+EXE =
```
Comment (thess) [inline multimedia/v4l2rtspserver/files/config.lede]: EOL missing

File: multimedia/v4l2rtspserver/files/v4l2rtspserver.config [hunk 2]
```diff
@@ -0,0 +1,9 @@
+config v4l2rtspserver 'core'
+	option enabled '1'
+	option device '/dev/video0'
+	option port '554'
+	option resolution '1024x960'
+	option fps '15'
+	option path 'stream'
+	option username ''
+	option password ''
```
Comment (thess) [inline multimedia/v4l2rtspserver/files/v4l2rtspserver.config]: EOL missing

File: multimedia/v4l2rtspserver/files/v4l2rtspserver.init [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+start_service() {
+	config_load "$SERVICE"
+	config_foreach start_instance "$SERVICE"
+}
+
+service_triggers() {
+	procd_add_reload_trigger "$SERVICE"
+}
```
Comment (thess) [inline multimedia/v4l2rtspserver/files/v4l2rtspserver.init]: EOL missing

**Comment by thess** [comment]
I tested this with a Logitech camera which outputs MJPG format. Apparently the default video input format is H264 and I got an error on server startup

>[ERROR] src/V4l2Device.cpp:166, Cannot set pixelformat to:H264 format is:MJPG

I added `-fMJPG` option to the startup command and the stream came up OK. Please add the format option to the config and startup files.

**Comment by roger-** [comment]
Sure, pull updated.

**Comment by thess** [comment]
If you want to pass Travis CI checks, you really should update your git profile with your full name so Author matches SoB.
```$ git config --global user.name "Roger Dammit"```

Then for this case to fix PR and toggle Travis again...
```$ git commit --amend --reset-author --no-edit```
```$ git push -f```

**Comment by roger-** [comment]
Thanks, let me fix that in the next iteration.

**Comment by thess** [comment]
OK - this time :wink:

**Pull request merged**